test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/IncludeSearchTests.cs

```diff
@@ -273,6 +273,17 @@ public async Task GivenAnIncludeSearchExpression_WhenSearched_DoesNotIncludeUnty
                 Fixture.ObservationWithUntypedReferences);
         }
 
+        [Fact]
+        public async Task GivenAnIncludeSearchExpressionWithAllResources_WhenSearched_DoesNotIncludeUntypedReferences()
+        {
+            string query = $"_id={Fixture.ObservationWithUntypedReferences.Id}&_include=*";
```

Comment:
What does _include=:* do now, is it the same?
Also do we have tests for _include=Observation:* too? (i.e. the other permutations of this syntax)

Reply:
Good questions!  The behavior between the two syntax is the same.  We just don't error out if the : is present,.
Yes, we do have a test for the other syntax, the new test case I added was modified from the test with the :